chainer/utils/imgproc.py

```diff
@@ -0,0 +1,31 @@
+import numpy
+
+
+def oversample(images, crop_dims):
```

Comment:
I simply use this word because caffe already use this word
(see https://github.com/BVLC/caffe/blob/master/python/caffe/classifier.py and https://github.com/BVLC/caffe/blob/master/python/caffe/io.py).